README.md

```diff
@@ -1,3 +1,16 @@
-# CP1404 Practicals
+# CP1404 Practicals - Lucas O'Brien
+
+## About this repo
+
+I am a first year IT student at JCU in Townsville.
+This is a repo of all my practical work for CP1404 - Programming II.
+
+3 Lessons of 'Clean Code' I've learnt
+
+- Lesson 1
```

Comment:
Maybe forgot to finish the lessons?

Reply:
Or is it related to prac_07 README.md, not sure

Reply:
Yeah, I've updated the Prac_07 README.md not the repo README.md, fixed now